### ISSUE TITLE
Improved Pending Transaction Support

### ISSUE THREAD

**Issue by dOrgJelli** [dOrgTech/DAOcreator#38]
Improve how the app handles pending transactions. Currently we store  a single state within the redux store telling us that we're waiting on a transaction (see interface WaitingAnimation in AppState.ts).

Ideally we would be able to support:
- [ ] Multiple Pending Transactions
- [ ] App Reloads W/ Stored State
- [ ] App Reloads W/O Stored State (out of scope for this task, requires watching for chain updates proactively)

**Comment by orishim** [comment]
Is this issue up to date? Do we still have the same issue with pending transactions?

**Comment by orishim** [comment]
Subsumed under [200](https://github.com/dOrgTech/DAOcreator/issues/200)
